### PR TITLE
feat: seed entraid connector

### PR DESCRIPTION
**Pull request by andreacappadona17**
[comment]: <> (This file has been added on OGCIO fork)
### Ticket:

- [19987](https://dev.azure.com/OGCIO-Digital-Services/Digital%20Services%20Programme/_workitems/edit/19987)

### Description

Seed data for OGCIO EntraID connector.

## Type

- [ ] **Dependency upgrade**
- [ ] **Bug fix**
- [ ] **New feature**
- [ ] **Dev change**

### Checklist:

- [ ] I have added tests that prove my fix is effective or that my feature works

### Screenshots:

N/A

### CODE REVIEW

File: packages/cli/src/commands/database/ogcio/ogcio-seeder-local.json [hunk 1]
```diff
@@ -331,6 +315,20 @@
           },
           "target": "MyGovId (MyGovId connector)"
         }
+      },
+      {
+        "id": "ogcio-entraid",
+        "sync_profile": true,
+        "connector_id": "ogcio-entraid",
+        "config": {
+          "clientId": "<client-id>",
```
Comment (SamSalvatico) [inline packages/cli/src/commands/database/ogcio/ogcio-seeder-local.json:324]: In the local version you can set a client id and secret directly, without the `<` 

Like in https://github.com/ogcio/logto/blob/6e4e333cbf6f6e4f087027b7f582e1d554c414aa/packages/cli/src/commands/database/ogcio/ogcio-seeder-local.json#L301

File: packages/cli/src/commands/database/ogcio/ogcio-seeder-local.json [hunk 2]
```diff
@@ -331,6 +315,20 @@
...
+          "tenantId": "organizations",
```
Comment (SamSalvatico) [inline packages/cli/src/commands/database/ogcio/ogcio-seeder-local.json:325]: Where is this tenantId from?
Usually it is "default"

File: packages/cli/src/commands/database/ogcio/ogcio-seeder.json [hunk 3]
```diff
@@ -319,9 +323,7 @@
           "password": false,
           "identifiers": []
         },
-        "social_sign_in_connector_targets": [
-          "MyGovId (MyGovId connector)"
-        ],
+        "social_sign_in_connector_targets": ["MyGovId (MyGovId connector)"],
```
Comment (SamSalvatico) [inline packages/cli/src/commands/database/ogcio/ogcio-seeder.json:326]: Do you need to add entra here?

**Review by SamSalvatico — COMMENTED** [review]
Remember to add the secrets in the `terraform` repo

File: packages/cli/src/commands/database/ogcio/ogcio-seeder-local.json (hunk 2 above)
Comment (andreacappadona17) [reply packages/cli/src/commands/database/ogcio/ogcio-seeder-local.json:325]: For EntraID the organizations tenant is used to enable access to users from multiple tenants/organizations.

**Review by edge33 — APPROVED** [review]
LGTM

File: packages/cli/src/commands/database/ogcio/ogcio-seeder.json (hunk 3 above)
Comment (andreacappadona17) [reply packages/cli/src/commands/database/ogcio/ogcio-seeder.json]: Updated

**Comment by andreacappadona17** [comment]
PR to add terraform secrets https://github.com/ogcio/life-events-terraform/pull/86

**Review by SamSalvatico — APPROVED** [review]

**Pull request merged**